examples/apps/echo/client/echo_client.py

```diff
@@ -65,6 +65,11 @@ def ParseCommandLine(args):
         help="the Ethereum address of the registry list",
         type=str)
 
+    use_service.add_argument(
+        "-k", "--key",
```

Comment:
I'm not certain that we need to add key as argument to generic client. That way why not add ethereum account also as argument.  I was thinking we could add blockchain specific configuration in config file rather than adding it as argument to generic client. Let's discuss this.

Reply:
We were handling the key differently(passing through envt variables) probably because we did not want to put it in some file(config file in this case). On that basis, passing it at run time would provide some security.